sklearn/cross_decomposition/tests/test_pls.py

```diff
@@ -404,12 +404,12 @@ def test_copy(Est):
 
     X_orig = X.copy()
     with pytest.raises(AssertionError):
-        pls.transform(X, Y, copy=False),
+        (pls.transform(X, Y, copy=False),)
```

Comment:
Won't this work by simply calling `pls.transform` without wrapping it within a tuple?

Reply:
Yea. I do not know how this change got into this PR. 😅